packages/cf-style-provider/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "cf-style-provider",
+  "description": "Cloudflare Style Provider",
+  "version": "0.1.0",
+  "main": "lib/index.js",
+  "module": "es/index.js",
+  "jsnext:main": "es/index.js",
+  "author": "Vojtech Miksu <vojtech@cloudflare.com>",
+  "license": "BSD-3-Clause",
+  "publishConfig": {
+    "registry": "http://registry.npmjs.org/"
+  },
+  "dependencies": {
+    "cf-style-const": "^0.3.0",
+    "cf-style-container": "^0.2.0",
+    "fela": "^4.3.2",
```

Comment:
Root package.json? Do you mean cf-ui/package.json? I don't think so. 

Pretty much the whole point of cf-style-provider is to shield you from setting up Fela if you want to consume cf-ui components and provide you with all necessary dependencies.